### DISCUSSION TITLE
V4 Notary Election: Proposed Tier System

### DISCUSSION THREAD

**Discussion by Carohere** [filecoin-project/notary-governance#623] [General]
The concept of ‘Tiers’ for Notaries. There would be Tiers of participation, where
1 = Notary who votes/proposes
2 = Notary who approves Direct Allocations
3 = Notary who approves LDNs

As the tier system is still in the design stage, any ideas are welcome!

![image](https://user-images.githubusercontent.com/106579285/193903038-25f804b2-1ea5-492b-b2ea-15519d571679.png)

**Comment by Carohere** [comment]
A few key points that I think need to be discussed:

- will there be different criteria applied to candidates for each tier?

- will notaries in different tiers hold different influences? 
I assume so. Since Datacap is what matters most to community members after all. At this point, I don't see much difference between proposing as a Tier 1 notary and as a community participant 
>1 = Notary who votes/proposes
- what is the scope of voting?
I haven't seen any votes other than on FIP0036. The range of notaries voting needs to be defined.

**Comment by Kevin-FF-USA** [comment]
Hi @Carohere ,
We are going to publish a formalize proposal to the scoring rubric shortly. Ok if you want to have this open, but it may get confusing to have multiple conversations at one time on the same topic.

**Reply by Carohere** [reply]
Hey Kevin, thanks for letting me know. I'm going to keep this up until the proposal is made official. Just in case someone wants to share ideas before it's released.

**Reply by jamerduhgamer** [reply]
I agree with Kevin's point on having a singular point to discuss a topic. Will definitely bring this up for discussion during the next notary meeting!

@Carohere, I like the idea of having a tiered / defined roles for notaries 🙂

**Reply by Carohere** [reply]
Hi james,I like it too! The idea is brilliant and we should definitely have more in-depth discussions about this. I'll be looking forward to hearing your thoughts soon! 😊 And once the official proposal is made, I'll close this discussion and move all comments there so don't worry about it.
